week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,9 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  return str.slice(0, 1).toUpperCase() + str.slice(1, str.length).toLowerCase();
```

Comment:
You probably don't need to do the `toLowerCase()` on the remainder of the string. The requirements only say that you should capitalise the first letter.